packages/avado/dappnode_package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hopr.avado.dnp.dappnode.eth",
-  "version": "0.100.0",
+  "version": "1.69.3",
```

Comment:
We're always having this conflict. Is there an opportunity to get rid of this?

Reply:
Sadly no, that's an AVADO hack, for now, to ensure we always build a new version on master.